src/main/resources/application.yml

```diff
@@ -1,15 +1,16 @@
-openmrs.root.url: https://localhost.localdomain/openmrs/ws/rest/v1
-bahmni.login.url: https://localhost.localdomain/bahmni/home/#/login?showLoginMessage
-reports.url: https://localhost.localdomain/bahmnireports/report
+openmrs.root.url: http://localhost/openmrs/ws/rest/v1
+bahmni.login.url: http://localhost/bahmni/home/#/login?showLoginMessage
+reports.url: http://192.168.33.10/bahmnireports/report
 reports.json: /var/www/bahmni_config/openmrs/apps/reports/reports.json
-dhis.config.directory: /var/www/bahmni_config/dhis2/
-dhis.url: dhisinstanceurl
-dhis.user: user
-dhis.password: password
-openmrs.db.url: jdbc:mysql://localhost.localdomain/openmrs?user=root&password=password
-openelis.db.url: jdbc:mysql://localhost.localdomain/openmrs?user=dbuser&password=dbpassword
+dhis.config.directory: /var/www/bahmni_config/dhis2/charikot/
+dhis.url: http://13.232.186.115:8080/
+dhis.user: admin
+dhis.password: Twit@512!
+openmrs.db.url: jdbc:mysql://192.168.33.10/openmrs?user=root&password=P@ssw03d
+openelis.db.url: jdbc:postgresql://localhost/clinlims?user=clinlims&password=clinlims
+submission.audit.folder: /dhis-integration-data
 log4j.config.file: log4j.properties
 server.port: 8040
 server.context-path: /dhis-integration/
-submission.audit.folder: /dhis-integration-data
-dhis.imam.dataset.id: UTGrrHEFwtt
\ No newline at end of file
+dhis.imam.dataset.id: UTGrrHEFwtt
+dhis.familyPlanning.dataset.id: JOAyT4yOEcs
```

Comment:
@suruchee  | @dipakthapaofficial, lets put this program specific configuration in the program specific configuration files.